test/integration/utility.py

```diff
@@ -1,67 +1,20 @@
+"""Packages utility methods for tests."""
```

Comment:
(nit, pre-existing) We may want to choose a better name, "utility" is a catch all and doesn't help the reader much. Even something like "test_utility" would be better.